drivers/storage/ide/pciidex/miniport.c

```diff
@@ -90,6 +93,30 @@ PciIdeXPnpDispatch(
 		return PciIdeXPdoPnpDispatch(DeviceObject, Irp);
 }
 
+/*!\brief Prints the given string with printf-like formatting to the kernel debugger.
+ * \param[in] DebugPrintLevel  Level of the debug message.
+ *                             Valid values are between 0 (Error) and 3 (Trace).
+ * \param[in] DebugMessage     Format of the string/arguments.
+ * \param[in] ...              Variable number of arguments matching the format
+ *                             specified in \a DebugMessage.
+ * \sa PciIdeDebug
+ */
+VOID
+PciIdeXDebugPrint(
+    _In_ ULONG DebugPrintLevel,
+    _In_z_ _Printf_format_string_ PCCHAR DebugMessage,
+    ...)
+{
+    va_list ap;
+
+    if (DebugPrintLevel <= PciIdeDebug)
+        DebugPrintLevel = 0;
```

Comment:
Add a comment to explain what this block is doing?

Reply:
Well, then, if I guess correctly, why not rather early return here and use 0 directly in the following call?
And use/add `#define DPFLTR_ERROR_LEVEL 0`, instead of a magic value.